.github/workflows/CI.yml

```
@@ -32,27 +32,24 @@ jobs:
         path: "OpenCoarrays-2.9.2/"
         key: ${{ steps.time.outputs.time }}
 
-    - name: Install Dependencies
+    - name: Install gfortran, OpenCoarrays, and fpm
       run: |
         sudo apt install -y gfortran-${GCC_V} wget python-dev python build-essential graphviz
         sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-${GCC_V} 100
         if [ ! -d OpenCoarrays-2.9.2 ] ; then wget -P . https://github.com/sourceryinstitute/OpenCoarrays/releases/download/2.9.2/OpenCoarrays-2.9.2.tar.gz && tar -xf OpenCoarrays-2.9.2.tar.gz && cd OpenCoarrays-2.9.2 && TERM=xterm ./install.sh -y; fi
         wget -P . "https://bootstrap.pypa.io/get-pip.py" && sudo python get-pip.py && rm get-pip.py
         sudo pip install ford
+        wget https://github.com/fortran-lang/fpm/releases/download/v0.2.0/fpm-0.2.0-linux-x86_64 && s    udo cp fpm-0.2.0-linux-x86_64 /usr/local/bin/fpm && sudo chmod a+x /usr/local/bin/fpm
```

```suggestion
        wget https://github.com/fortran-lang/fpm/releases/download/v0.2.0/fpm-0.2.0-linux-x86_64 && sudo cp fpm-0.2.0-linux-x86_64 /usr/local/bin/fpm && sudo chmod a+x /usr/local/bin/fpm
```

found your problem. just some errant white space snuck in